### ISSUE TITLE
Images have alternate text that can be read by screen reader software

### ISSUE THREAD

**Issue by CristinaSolana** [envistaInteractive/itagroup-ecommerce-template#219]
27 Missing alernative text.  Currently caused by template state with no images on the site.
